sdk-core/src/test/java/care/data4life/sdk/RecordServiceTest.kt

```diff
@@ -823,6 +825,79 @@ class RecordServiceTest : RecordServiceTestBase() {
         inOrder.verifyNoMoreInteractions()
     }
 
+    @Test
+    @Throws(DataRestrictionException.UnsupportedFileType::class, DataRestrictionException.MaxDataSizeViolation::class)
+    fun `Given, checkDataRestrictions is called, with a Resource, which has non extractable Attachments, it returns without a failure`() {
+        // Given
+        val resourceStr = "{\n" +
+                "    \"resourceType\": \"Patient\",\n" +
```

Comment:
I think the example payload should be shrinked to a minimal version, otherwise there is a lot of information that is not related to this check

Reply:
or the resource is moved to the resource folder

Reply:
I pulled it out in the resource folder...However, since we will refactor the test soonish, I will invest a bit more time to think of a more generic example then, to decouple the test.